argilla-frontend/translation/en.js

```diff
@@ -13,6 +13,7 @@ export default {
   copied: "Copied",
   copyClipboard: "Copy to clipboard",
   copyLink: "Copy link",
+  copyRecord: "Copy record",
```

Comment:
Remember that now we have translations for German, so you should update the de.js file.